code/modules/projectiles/guns/magic/wand.dm

```diff
@@ -10,6 +10,10 @@
 	var/variable_charges = TRUE
 
 /obj/item/gun/magic/wand/Initialize()
+	if(type == /obj/item/gun/magic/wand)
+		visible_message("<span class='warning'>[src] disappears!</span>")
+		qdel(src)
+		return

```

Comment:
admins should still be able to spawn the base wand if they want to edit it.

should just remove it, you've already made it obvious from the wand of nothing no need for quirky checks.

Reply:
okay